CHARTER.adoc

```diff
@@ -1,9 +1,14 @@
+= Charter of The TODO Group
+
+:toc:
+
  . *Mission of the TODO Group (“TODO”).*
  The TODO Group’s mission is to identify key policy and process choices related to corporate open source engagement and create tools and educational materials that promote best practices around such engagements.
```

Comment:
```suggestion
 The TODO Group’s mission is to provide a community of practice and a structured way to identify key policies and processes related to organization's open source implementation and create tools and educational resources to aid them across its journey.
```

Reply:
> 

The TODO Group's mission is to convene a community of practice focused on best practices for corporate open source programs. The Community of Practice will identify, document, and share policy and process best practices and create tools and educational materials. The group serves business professionals in OSPOs looking to collaborate with fellow TODO members and other related affinity groups and communities, including but not limited to NGOs, NPOs, educational institutions, or governments.

Reply:
I do not agree. I am explicitly stating that TODO serves corporate members that collaborate with all these other groups.

Reply:
final resolution from the SC members (5/7) during SC Charter call: update scope using @lhawthorn last definition:


*The TODO Group's mission is to convene a community of practice focused on best practices for corporate open source programs. The Community of Practice will identify, document, and share policy and process best practices and create tools and educational materials. The group serves business professionals in OSPOs looking to collaborate with fellow TODO members and other related affinity groups and communities, including but not limited to NGOs, NPOs, educational institutions, or governments.*

Reply:
My thoughts here as we discussed the topic of TODO Group and having OSPOs outside the corporate world join the organization a few years ago when we posted this blog + FAQ:

https://todogroup.org/blog/becoming-a-todo-group-member-as-an-associate/
https://todogroup.org/faq/#:~:text=about%20membership%20benefits.-,As%20a%20university%20or%20academic%20institution%2C%20can%20I%20join%20the,discussions%20with%20experienced%20OSPO%20professionals.

The charter should reflect that we FOCUS on corporate OSPOs but don't exclude folks on joining the organization from outside the corporate world, we already have governments and academic institutions involved as official members.

Reply:
What about something like:

*The TODO Group's mission is to convene a community of practice focused on best practices for open source programs. While our primary focus is on corporate OSPOs, we welcome members from outside the corporate world, including governments, NGOs, NPOs, educational institutions, and others. The Community of Practice will identify, document, and share policy and process best practices and create tools and educational materials. The general members serve professionals in OSPOs looking to collaborate with fellow TODO general members and other related affinity groups and communities. Our general members include a diverse group of organizations committed to advancing open source best practices and creating a more inclusive and collaborative open source ecosystem.*

Reply:
From the last SC meeting: 

*leave the statement as it was initially. Update the section about the general membership*